tests/home.nix

```diff
@@ -34,9 +38,6 @@
       delta.enable = true;
     };
     gitui.enable = true;
-    # this is enabled by default already, but still
-    # listing explicitly so we know it's tested
-    glamour.catppuccin.enable = true;
```

Comment:
We should keep using this attribute in our tests to make sure the alias works (similar to the others)